### PR TITLE
bun 1.0.3

### PR DESCRIPTION
**Pull request by cdmistman**
Why
===

like recent bun

What changed
============

- update bun to 1.0.3

Test plan
=========

- `bun --version` with module installed prints 1.0.3

Rollout
=======

_Describe any procedures or requirements needed to roll this out safely (or check the box below)_

- [x] This is fully backward and forward compatible

### CODE REVIEW

**Review by ryantm — CHANGES_REQUESTED** [review]
We haven't fixed the runner yet, so please do not merge this.

**Comment by cdmistman** [comment]
[13b1054](https://github.com/replit/nixmodules/pull/125/commits/13b105484879a2c8101ee2c4ff0e4c682529aee7) reverted the module config to use the package-json-runner script/config from v3. the only delta is the path for the bun derivation

File: pkgs/modules/bun/default.nix [hunk 1]
```diff
@@ -6,17 +6,30 @@ let
...
+    fi
+  '';
 in
 
 {
   id = "bun-${community-version}";
   name = "Bun Tools";
 
   imports = [
-    (import ../run-package-json {
```
Comment (ryantm) [inline pkgs/modules/bun/default.nix:16]: Are you able to delete this directory now too?

File: pkgs/modules/bun/default.nix [hunk 2]
```diff
@@ -6,17 +6,30 @@ let
...
+    elif module=$(${pkgs.jq}/bin/jq -r 'if .type == "module" then .module // empty else empty end' package.json 2>/dev/null); [[ -n $module ]]; then
+      echo "+ bun $module";
+      ${bun}/bin/bun $module;
+    elif main=$(${pkgs.jq}/bin/jq -r '.main // empty' package.json 2>/dev/null); [[ -n $main ]]; then
+      echo "+ bun $main";
+      ${bun}/bin/bun $main;
+    elif [[ -n $file ]]; then
+      echo "+ bun $file";
+      ${bun}/bin/bun $file;
+    fi
```
Comment (ryantm) [inline pkgs/modules/bun/default.nix:24]: What about the `replit-dev` runner from before?

File: pkgs/modules/bun/default.nix (hunk 1 above)
Comment (cdmistman) [reply pkgs/modules/bun/default.nix:16]: i still plan on fixing `run-package-json`, i just want to do that in a different pr. the script in this pr is the v3 runner script

**Review by ryantm — APPROVED** [review]

**Pull request merged**